tests/manage/monitoring/conftest.py

```diff
@@ -342,3 +347,150 @@ def stop_osd():
     oc.exec_oc_cmd(f"scale --replicas=1 deployment/{osd_to_stop}")
 
     return measured_op
+
+
+def create_dummy_osd(deployment):
```

Comment:
Since it's not a fixture, thinking it would be better to have this function in another file, like helpers.py

Reply:
No, please don't add functions into helpers file withing a test directory tree.

Reply:
You could add it into ocsci module.